infrastructure/outputs.tf

```diff
@@ -59,3 +59,14 @@ output "service_account" {
 output "private_cluster" {
   value = var.private_cluster
 }
+
+
+output "service_account" {
```

Comment:
You have another `service_account` output variable declared above what's the difference between these two? The build is failing because of this `Error: Duplicate output definition`

Reply:
My apologies. The same output was introduced by the other guide that is already merged to the repo. It seems the merge has produced duplicates.